packagedb/package_managers.py

```diff
@@ -0,0 +1,635 @@
+#
```

Comment:
Bring the tests from https://github.com/nexB/vulnerablecode/blob/main/vulnerabilities/tests/test_package_managers.py over